assets/models/system/bert-large-cased/description.md

```diff
@@ -63,6 +63,13 @@ You can use the raw model for either masked language modeling or next sentence p
 
 Note that this model is primarily aimed at being fine-tuned on tasks that use the whole sentence (potentially masked) to make decisions, such as sequence classification, token classification or question answering. For tasks such as text generation you should look at model like GPT2.
 
+# Inference samples
+
+Inference type|Python sample (Notebook)
+|--|--|
+Real time|[sdk-example.ipynb](https://aka.ms/sdk-notebook-examples)
+Real time|[fill-mask-online-endpoint.ipynb](https://aka.ms/fill-mask-online-endpoint-oss)
```

Comment:
we can follow evaluation and FT sample example and add datasets and use case as well, so all samples are consistent

Reply:
TODO in next PRs if needed